### PR TITLE
Change from one-by-one skill registration to bulk processing

### PR DESCRIPTION
**Pull request by idegtiarov**

### CODE REVIEW

File: edxadapt/static/js/src/edxadapt.js [hunk 1]
```diff
@@ -28,31 +28,25 @@ function EdxAdaptXBlock(runtime, element) {
...
-                })
-                .done(function() {
-                    console.log("Skill " + skill + "is  configured");
-                })
-                .fail(function() {
-                    console.log('Failed to configure skill "'+skill+'" for user "'+studentId+'"');
-                    setStatusNok();
-                });
-            )
+        var student_config = {
```
Comment (z4y4ts) [inline edxadapt/static/js/src/edxadapt.js]: Use camelCase

File: edxadapt/static/js/src/edxadapt.js [hunk 2]
```diff
@@ -28,31 +28,25 @@ function EdxAdaptXBlock(runtime, element) {
...
+            'course_id': courseId,
+            'params': params,
+            'user_id': studentId,
+            'skills_list': skills
+        };
+        $.ajax({
```
Comment (z4y4ts) [inline edxadapt/static/js/src/edxadapt.js]: I guess you want to return this deferred.

**Review by z4y4ts — COMMENTED** [review]
Couple of things before I can give green light.

File: edxadapt/static/js/src/edxadapt.js (hunk 2 above)
Comment (idegtiarov) [reply edxadapt/static/js/src/edxadapt.js]: Yes sure, fixed

File: edxadapt/static/js/src/edxadapt.js (hunk 1 above)
Comment (idegtiarov) [reply edxadapt/static/js/src/edxadapt.js]: done

**Review by z4y4ts — APPROVED** [review]

**Pull request merged**